include/openssl/ssl.h

```diff
@@ -1737,8 +1743,8 @@ void SSL_free(SSL *ssl);
 __owur int SSL_waiting_for_async(SSL *s);
 __owur int SSL_get_all_async_fds(SSL *s, OSSL_ASYNC_FD *fds, size_t *numfds);
 __owur int SSL_get_changed_async_fds(SSL *s, OSSL_ASYNC_FD *addfd,
-                                     size_t *numaddfds, OSSL_ASYNC_FD *delfd,
-                                     size_t *numdelfds);
+                                     size_t *numaddfds,
+                                     OSSL_ASYNC_FD *delfd, size_t *numdelfds);
```

Comment:
Going from column 77 (old) to column 78 (new)?  What's the motivation for this change?

Reply:
indent tool choice may be, I don't remenber